modules/core/src/main/java/org/apache/ignite/internal/processors/configuration/distributed/DistributedConfigurationProcessor.java

```diff
@@ -155,6 +157,29 @@ private static String toPropertyKey(String metaStorageKey) {
         return prop;
     }
 
+    /**
+     * @return Public properties.
+     */
+    public List<DistributedChangeableProperty<Serializable>> properties() {
+        return props.values().stream()
+            .filter(p -> p instanceof DistributedChangeableProperty)
```

Comment:
How props ( Map<String, DistributedChangeableProperty> props ) can contain a value of any other type?